README.md

```diff
@@ -50,7 +50,7 @@ toolkit](https://software.intel.com/en-us/openvino-toolkit). For example, users
   - Update `find` command to find configurations of tasks/algorithms
   - Introduce `build` command to customize task or model configurations
   - Automatic algorihm selection for the `train` command using the given input dataset
-- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+- Adaptation of [datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
```

Comment:
Do we need this change?
![image](https://user-images.githubusercontent.com/4629606/211229542-848ac1d5-bfb1-42c6-9cf1-3793bc01b582.png)

Reply:
Then how about to change `Datumaro` for `datumaro` in README.md?

Reply:
@wonjuleee Could you give us some advice? `Datumaro` or `datumaro`?

Reply:
> Then how about to change `Datumaro` for `datumaro` in README.md?

Sorry, I cannot understand. Do you mean that we need to replace all `datumaro` by `Datumaro`?

Reply:
Wonju said `Datumaro` is right :)